ycmd/tests/get_completions_test.py

```diff
@@ -115,9 +123,800 @@ def GetCompletions_IdentifierCompleter_WorksForSpecialIdentifierChars_test():
                            completion_data ).json[ 'completions' ]
 
   assert_that( results,
-               has_items( CompletionEntryMatcher( 'font-size' ),
-                          CompletionEntryMatcher( 'font-family' ) ) )
+               has_items( CompletionEntryMatcher( 'font-size', '[ID]' ),
+                          CompletionEntryMatcher( 'font-family', '[ID]' ) ) )
+
 
+@with_setup( Setup )
+def GetCompletions_Fall_Back_ClangCompleter_test():
+
+  # because we use it a lot
+  NO_COMPLETIONS_ERROR = ErrorMatcher( RuntimeError, NO_COMPLETIONS_MESSAGE )
+
+  """
+  GetCompletions_Fall_Back_test
+
+  Tests for:
+    - [X] multiple languages: (use cases)
+      - [X] C:      macros
+      - [X] C++:    templates
+      - [X] Python: properties set from external method
+    - [X] Forced vs. Triggered semantic completion
+    - [X] Identifiers tagged with '[ID]'
+    - [X] Filter/sort based on query
+    - [X] Min chars for completion (only applies to ID)
+    - [X] Extra conf throws exception
```

Comment:
Done. I've trimmed the tests to just one each covering each relevant (IMO) code path.